src/App.tsx

```diff
@@ -1,61 +1,122 @@
+import { useState } from 'react';
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList/TodoList';
 
 export const App = () => {
+  const [title, setTitle] = useState('');
+  const [userName, setUserName] = useState('');
+  const [todoList, setTodoList] = useState(todosFromServer);
+  const [isWarningTitle, setIsWarningTitle] = useState(false);
+  const [isWarningUser, setIsWarningUser] = useState(false);
+  const handleSubmit = (e: React.FormEvent) => {
+    e.preventDefault();
+
+    let status = false;
+
+    if (!userName) {
+      status = true;
+
+      setIsWarningUser(true);
+    }
+
+    if (!title.trim() || !title) {
+      status = true;
+
+      setTitle('');
+      setIsWarningTitle(true);
+    }
+
+    if (status) {
+      return;
+    }
+
+    const user = [...usersFromServer].find(
+      (userItem) => userItem.name === userName,
+    );
+
+    if (!user) {
+      throw new Error('Submit error, user not defined');
+    }
+
+    if (user) {
+      const todo = {
+        id: todoList.length,
```

Comment:
Change 'completed: true' to 'completed: false' since new todos should initially be uncompleted.